### PR TITLE
Added calyptia version subcommand

### LINKED ISSUE #231: Modify version information output
# New feature request

## Describe the proposal

Reduce the output from `calyptia --version` to just be the version number.
This will be more useful for automation where we just want to compare a version.

### Checklist for proposal reviewer

- [ ] The proposal is clear on what changes need to be made in the code base.

- [ ] The proposal is clear on the architectural changes that will be affected.

- [ ] The proposal is clear on the breaking changes and how to mitigate them.

- [ ] The proposal is clear on what the deployment/configuration strategy is.

- [ ] The proposal is clean on how testing will validate the feature implementation.

### Reviewers

- [ ] Reviewed by a team member of the @calyptia/backend team
- [ ] Reviewed by a product owner.

### Disclaimers

- If there are multiple solutions, please present each one separately.
- Save comparisons for the very end.
- Please do not create a Pull Request without creating an issue first.
- Any change needs to be discussed before proceeding.
- Failure to do so may result in the rejection of the pull request.

### PR DESCRIPTION
**Pull request by jasmingacic**
```
# calyptia version 
Version v1.0.1-23-gfdac727
```
Fixes #231 

@patrick-stephens is this what you had in mind with https://github.com/calyptia/cli/issues/231
If you prefer we can modify this maybe strip Version string and just print out actual tag.

### CODE REVIEW

File: cmd/version/version.go [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+
 var (
 	DefaultCloudURLStr = "https://cloud-api.calyptia.com"
 	Version            = "dev" // To be injected at build time:  -ldflags="-X 'github.com/calyptia/cli/cmd/version.Version=xxx'"
 )
+
+func NewVersionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "version",
+		Short:         "v",
```
Comment (nicolasparada) [inline cmd/version/version.go:17]:
```suggestion
		Aliases:        []string{"v"},
```

"Short" means a short description, not an alias.

File: cmd/version/version.go [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+		SilenceErrors: true,
```
Comment (nicolasparada) [inline cmd/version/version.go:18]:
```suggestion
```

Why?

File: cmd/version/version.go [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+		SilenceUsage:  true,
```
Comment (nicolasparada) [inline cmd/version/version.go:19]:
```suggestion
```

**Review by nicolasparada — CHANGES_REQUESTED** [review]
Left some comments.

File: cmd/version/version.go (hunk 1 above)
Comment (jasmingacic) [reply cmd/version/version.go]: Actually no need for short hand "v" using full subcommand name is enough.

**Review by nicolasparada — APPROVED** [review]

**Pull request merged**